lib/class-wp-theme-json.php

```diff
@@ -271,25 +250,43 @@ class WP_Theme_JSON {
 		),
 	);
 
+	const ELEMENTS = array(
+		'link' => 'a',
+		'h1'   => 'h1',
+		'h2'   => 'h2',
```

Comment:
link and h1 are both elements but while h1 reacts like a block where one can control everything the link element just manifests as a UI artifact inside the block (Link color) should we differentiate between the two types of elements?

Reply:
Related https://github.com/WordPress/gutenberg/pull/30541#discussion_r621978442 TLDR: UI for elements is still a work in progress and this PR doesn't modify it (the link color is still expressed as another panel within color but going forward it'll have more design surface such as typography, background color, etc).